### PR TITLE
new package: com.google.android.apps.maps

### PR DESCRIPTION
**Pull request by sevenrock**

### CODE REVIEW

File: modules/Maps/Android.mk [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Auto-generated Makefile from OpenGapps. Do not modify.
+LOCAL_PATH := .
+include $(CLEAR_VARS)
+LOCAL_MODULE := Maps
+LOCAL_PACKAGE_NAME := com.google.android.apps.maps
+LOCAL_MODULE_PATH := $(PRODUCT_OUT)/system/priv-app
```
Comment (mfonville) [inline modules/Maps/Android.mk]: Shouldn't this be in app instead of priv-app?

**Comment by sevenrock** [comment]
patch updated, thanks for the review

**Comment by Blystad** [comment]
Hey,

Thank you for your contribution.

Sorry for not attending to your pull request. I forgot to set the correct "Watch" setting on Github.

I'll test your Pull Request tomorrow, and get back to you.

**Comment by Blystad** [comment]
Thanks for your contribution. LGTM.

**Pull request merged**